api/src/iot/import_utils_apis.py

```diff
@@ -479,6 +482,50 @@ def parse_beweegbare_fysieke_afsluiting(data: dict) -> Generator[SensorData, Non
             )
 
 
+def parse_anpr(data: dict) -> Generator[SensorData, None, None]:
+    """
+    convert the anpr data dict into PersonData en
+    SensorData objects and yield it.
+    """
+    # create the personData from static defined data because the api doesn't provide
+    # all the required fields
+
+    person_data = PersonData(
+        organisation='Gemeente Amsterdam',
+        email='Meldingsplicht.Sensoren@amsterdam.nl',
+        telephone='14020',
+        website='https://www.amsterdam.nl/',
+        first_name='Afdeling',
+        last_name_affix='',
+        last_name='anpr management'
+    )
+
+    features = data['features']  # list of sensors i think for now
+    if features:
+        for feature in features:
+            geometry = feature['geometry']  # geometry dict
+            latitude = geometry['coordinates'][1]
+            longitude = geometry['coordinates'][0]
+            properties = feature['properties']  # properties dict
+            goals = properties['doel']  # a list of goals
+
+            yield SensorData(
+                owner=person_data,
+                reference=properties['id'],
+                type='Optische / camera sensor',
+                location=LatLong(latitude=latitude, longitude=longitude),
+                contains_pi_data='Nee',  # api has no contains_pi_data
```

Comment:
Dit moeten we checken - ANPR staat voor "Automatic number plate recognition". Ik ben vrijwel zeker dat contains_pi_data moet "Ja" zijn hier - maar we moeten checken welke legal ground erbij hoort.